Dockerfile

```diff
@@ -2,20 +2,26 @@ FROM balena/open-balena-base:v14.6.1
 
 EXPOSE 80
 
+# hadolint ignore=DL3008
 RUN apt-get update \
-	&& apt-get install \
+	&& apt-get install -y --no-install-recommends \
```

Comment:
Do you know if this results in anything not now being installed that we would actually want installed?

Reply:
I confirmed that in this case there is no difference.

```
#5 13.43 The following additional packages will be installed:
#5 13.43   libjemalloc2 liblua5.1-0 liblzf1 lua-bitop lua-cjson redis-tools
#5 13.43 Suggested packages:
#5 13.43   ruby-redis
#5 13.81 The following NEW packages will be installed:
#5 13.81   libjemalloc2 liblua5.1-0 liblzf1 lua-bitop lua-cjson musl redis-server
#5 13.81   redis-tools
```
```
#5 13.32 The following additional packages will be installed:
#5 13.32   libjemalloc2 liblua5.1-0 liblzf1 lua-bitop lua-cjson redis-tools
#5 13.32 Suggested packages:
#5 13.32   ruby-redis
#5 13.69 The following NEW packages will be installed:
#5 13.69   libjemalloc2 liblua5.1-0 liblzf1 lua-bitop lua-cjson musl redis-server
#5 13.69   redis-tools
```